github_changelog_generator.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency("github_api", ["~> 0.12"])
   spec.add_runtime_dependency("colorize", ["~> 0.7"])
+  spec.add_runtime_dependency("sqlite3")
```

Comment:
I'm not really familiar with JRuby, so it's not something I could test. It doesn't even have to be sqlite, I just needed something that would autocommit without too big of a performance hit (end preferably not a big heap of files).